tsconfig.json

```diff
@@ -14,7 +14,6 @@
     "module": "es2020",
     "paths": {
       "@skyux/tabs" : [
-        "dist/tabs",
```

Comment:
☝️ not part of this PR, but noticed we didn't catch this in migration. References should be resolved to `dist` in only very unique cases (like `@skyux/theme`).